src/app/modules/time-clock/components/entry-fields/entry-fields.component.spec.ts

```diff
@@ -167,19 +169,43 @@ describe('EntryFieldsComponent', () => {
     component.setDataToUpdate(entry);
     spyOn(toastrServiceStub, 'error');
 
-    const hourInThePast = moment().subtract(6, 'hour').format('HH:mm:ss');
+    const hourInThePast = moment().subtract(6, 'hour').format('HH:mm');
     component.entryForm.patchValue({ start_hour : hourInThePast});
     component.onUpdateStartHour();
 
     expect(toastrServiceStub.error).toHaveBeenCalled();
+    expect(component.showtimeInbuttons).toEqual(false);
+  });
+
+  it('should show time In buttons when time is modified', () => {
+    component.activeTimeInButtons();
+
+    expect(component.showtimeInbuttons).toEqual(true);
+  });
+
+  it('should go back to the starting time and disappear buttons', () => {
```

Comment:
The message is kind of confusing, maybe the following could work:
"should reset displayed time and hide control buttons when cancelTimeInUpdate"

Reply:
Sure, sound better it's changed, thanks.